src/centralized.rs

```diff
@@ -0,0 +1,499 @@
+//! ALBA's bounded DFS scheme using Blake2b as hash function.
```

Comment:
Suggest renaming the file to `centralized_telescope.rs` since we will have the simple lottery scheme as well.

Reply:
Sure.
Nitpicking: we could argue that the lottery scheme is both centralized and decentralized hence there is only one centralized scheme.